src/main/java/org/eclipse/yasson/internal/JsonbContext.java

```diff
@@ -130,4 +139,23 @@ public JsonbConfigProperties getConfigProperties() {
     public InstanceCreator getInstanceCreator() {
         return instanceCreator;
     }
+
+    private JsonbComponentInstanceCreator initComponentInstanceCreator(InstanceCreator instanceCreator) {
+        ServiceLoader<JsonbComponentInstanceCreator> loader = AccessController
```

Comment:
This also applies to further calls discovering Jsonp, but calling service loader slows down cold start of Jsonb (`JsonbBuilder.create()`). Do we want to add some classloader based cache here in future?

Reply:
Well, although it sounds like a good idea the caches are often tricky. And if I understand it correctly users are encouraged to resuse `Jsonb` and `JsonbBuilder` instances for optimal performance.

Reply:
Yes Jsonb instance is encouraged to be cached. If done, similar caching is not needed. However there is number of [reappearing similar requests](https://github.com/eclipse-ee4j/jsonp/issues/154) in Jsonp. Using APP classloader may solve it for both even when instances are not cached.

Reply:
I'm still not convinced that we should cache this but feel free to address this problem in another PR ;-)

Reply:
Sure, I am not asking to fix it here.